src/test/java/com/eppo/sdk/EppoClientTest.java

```diff
@@ -1,34 +1,33 @@
 package com.eppo.sdk;
 
+import static cloud.eppo.helpers.AssignmentTestCase.parseTestCaseFile;
```

Comment:
This is another file where I'd ignore the diff and just look at the new (green) code